src/javascripts/components/events.js

```diff
@@ -1,7 +1,37 @@
-const initEvents = () => {
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import eventsData from '../helpers/data/eventsData';
+
+import util from '../helpers/util';
+
+const displayEvents = (events) => {
+  let domString = '';
+  events.forEach((event) => {
+    domString += '<div class="card text-center">';
+    domString += '<h3 class="headTitle">My Events</h3>';
+    domString += `<h3 class="card-header">${event.name} - ${event.date}</h3>`;
+    domString += '<div class="card-body">';
+    domString += `<button class="btn btn-danger delete-events" id=${event.uid}>Delete</button>`;
```

Comment:
comment out this button - it will be added in another ticket

Reply:
done and